src/components/ImageUpload/ImageUpload.vue

```diff
@@ -0,0 +1,63 @@
+<template lang="pug">
+extends ../Base/InputBase.pug
+block input
+  .image-upload__wrapper
+  input.vfield__control.mb-3(
+    type="file",
+    :id="`${name}-image-upload`"
+    @change="onChangeImage"
+  )
+  .img-peview__wrapper(v-if="localValue && localValue.url")
+    .vfield__btn.vfield__btn--close(@click="removeImage(`${name}-image-upload`)")
+      i.bi.bi-x-circle
+    img.img-thumbnail(:src="localValue.url")
+</template>
+<script lang="ts" setup>
+import { useLocalValue, useFieldId, useFieldClass } from '@/utilities/hooks';
+
+interface Props {
```

Comment:
We should pass the endpoint to this component. Endpoint is backend URL where image uploaded